rfc/php_technical_committee.wiki

```diff
@@ -64,11 +64,12 @@ The TC Secretary is responsible for:
 
 The election process takes place annually, concurrent with the election of the Release Managers in approximately
 March/April.  (The exact date may vary year to year.)  The sitting TC Secretary officially begins the process with
-a call for nominations.  The nomination period lasts for one week.
+a call for nominations.  The nomination period lasts for one month.
```

Comment:
Doesn't a month make it longer than for the RMs?  If they're running concurrently then 2 weeks nomination, 2 weeks vote, for both seems reasonable.  Otherwise we're talking about a 6 week process to pick people, which is... excessive.

Reply:
RM nomination is month - at least it was month this year

Reply:
Really?  I wasn't even paying much attention to it...

I agree it should be aligned with the RM nominations, so I guess this is fine for now.  We may want to shorten both to 2 weeks but that would be for a future discussion.

Reply:
yeah